src/Graphql/Queries.js

```diff
@@ -2,7 +2,7 @@ import {gql} from '@apollo/client'
 
 export const GET_ALL_DONORS = gql `
   query {
-    queryUsers {
+    users {
```

Comment:
Had to change this to the key from the real API (query users was from the mock server).